### ISSUE TITLE
RTSP stream not possible anymore since 6.0.0

### ISSUE THREAD

**Issue by Xitro01** [fuatakgun/eufy_security#644]
**Describe the bug**

There is no option to let the integration use RTSP or P2P since 6.0.0. When clearing the P2P information it will just fill in the defaults after applying. After starting the stream it will not give me a stream (camera just shows as idle) and Stream Provider always shows P2P.

I don't have the RTSP add-on installed, as your new README says it's not needed if you use RTSP. RTSP has been more stable for me, so I want to try that out first in this new version.

**Comment by fuatakgun** [comment]
Use start_rtsp_livrstream / stop_rtsp_livestream calls rather than turn_on and turn_off

**Comment by Xitro01** [comment]
When I installed the RTSP add-on it works straight away by the way. But I get that distorted image again.

Using start/stop RTSP livestream service, gives me the error message: Camera (%s) does not support RTSP Stream

**Comment by fuatakgun** [comment]
So your camera doesn't support rtsp natively, integration generates the rtsp for you using p2p. You are actually using p2p based streaming.

What is video codec sensor saying?

**Comment by Xitro01** [comment]
> So your camera doesn't support rtsp natively, integration generates the rtsp for you using p2p. You are actually using p2p based streaming.
> 
> What is video codec sensor saying?

Video Codec
h264

**Comment by Xitro01** [comment]
> I don't know why it is distorted and don't have a solution for you.
> 
> You can lower all quality settings and video dimensions and try again after restarting.
> 
> At last, you can share your device with me and I can debug it

Already played with the quality/resolution settings.

It's the T8200, also see: https://github.com/fuatakgun/eufy_security/issues/434

I can try using Go2RTC, as you updated it that other RTSP servers can be used now. Let me try again, to see if I get that running, I know Alex also moved his WebRTC to Go2RTC now.

**Comment by fuatakgun** [comment]
I had bad experience with go2rtc with cameras like eufy, streaming on demand.

I tried go2rtc and other webrtc addon and nothing good came out of it.

**Comment by Xitro01** [comment]
I once installed the Go2RTC addon, added the Eufy camera to it and it worked flawless. But I couldn't integrate it in HA well so I stopped with that.

The 3.0 beta version of his integration now uses Go2RTC: https://github.com/AlexxIT/WebRTC
I just updated to his beta through HACS and now I no longer get a distorted image. So hopefully that permanently solved it!

**Comment by fuatakgun** [comment]
and you are using p2p streaming, right?

**Comment by Xitro01** [comment]
> and you are using p2p streaming, right?

Correct!
